### PR TITLE
Routines 

### LINKED ISSUE #54: User Routines
The user must:

- Create a new routine (exercise).
- Update an existing routine.
- Delete an existing routine.

### PR DESCRIPTION
**Pull request by angel23v**
## Quick Info

Now you can create a routine, see details, edit and delete, this is just the basis to later be able to create series within each routine.

## Screenshots
General view
index
New Routine
new
Edit Routine
edit
Routine Details
details

### CODE REVIEW

File: app/controllers/routines_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    @routine = Routine.new
+  end
+
+  def create
+    @routine = current_user.routines.build(post_params)
+    redirect_to routines_path if @routine.save
+  end
+
+  def index
+    # @routines = current_user.routines.order(created_at: :desc)
```
Comment (juperameza) [inline app/controllers/routines_controller.rb]: Don't leave commented code

Comment (MauricioNC) [inline app/controllers/routines_controller.rb]: Remove unnecessary comments

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/views/routines/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="">
+  <div class="">
+    <strong>Usuario:</strong> <%= @routine.user.name %>
+  </div>
+  <div class="">
+    <strong>Ejercicio:</strong> <%= @routine.exercise %>
+  </div>
+</div>
```
Comment (juperameza) [inline app/views/routines/show.html.erb]: Why do the divs have empty class attributes 
Just delete them 
Also mind the usage of divs 
It makes your seo worse and isn't accessible

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/controllers/routines_controller.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  def create
+    @routine = current_user.routines.build(post_params)
+    redirect_to routines_path if @routine.save
+  end
+
+  def index
+    @routines = Routine.all
+  end
+
+  def show
```
Comment (BlazeDex) [inline app/controllers/routines_controller.rb]: Change to def show;end

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -9,7 +9,9 @@
     <i class="bi bi-plus-square"></i>
   <% end %>
   <a href="">
```
Comment (BlazeDex) [inline app/views/layouts/_footer.html.erb]: Don't leave empty href. If you don't need it, delete it.

File: app/views/routines/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  <% @routines.each do |routine| %>
+      <h1>User:<%= routine.user.name %> Exercise:<%= routine.exercise %> </h1>
+    <div class="actions">
+        <%= link_to 'Edit Routine', edit_routine_path(routine.id), class: 'btn' %>
+        <%= link_to 'See Routine', routine_path(routine.id), class: 'btn' %>
+        <%= button_to 'Delete Routine', routine_path(routine.id), class: 'btn', method: :delete %>
+    </div>
+  <% end %>
+<% else %>
+  <p>No se encontraron rutinas.</p>
```
Comment (BlazeDex) [inline app/views/routines/index.html.erb]: Change to english. We need a consistent language.

File: app/views/routines/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<div>
+  <div>
+    <strong>Usuario:</strong> <%= @routine.user.name %>
```
Comment (BlazeDex) [inline app/views/routines/show.html.erb]: Same here.

**Review by MauricioNC — APPROVED** [review]

File: app/views/routines/index.html.erb (hunk 5 above)
Comment (angel23v) [reply app/views/routines/index.html.erb]: ¡¡¡¡that's right!!!!!

**Review by BlazeDex — APPROVED** [review]

**Review by juperameza — APPROVED** [review]

**Pull request merged**